server/routes.go

```diff
@@ -1190,11 +1192,11 @@ func Serve(ln net.Listener) error {
 		srvr.Close()
 		schedDone()
 		sched.unloadAllRunners()
-		gpu.Cleanup()
+		payloads.Cleanup(build.EmbedFS)
 		done()
 	}()
 
-	if err := llm.Init(); err != nil {
+	if _, err := payloads.RunnersDir(build.EmbedFS); err != nil {
```

Comment:
Example where we could have `runners.Dir()`

Also, does this code have a side effect? (the naming does not seem to imply this)

Reply:
Yes, this does have side effects
```
// Return the location where runners are stored
// If runners are payloads, this will either extract them
// or refresh them if any have disappeared due to tmp cleaners
```